### PR TITLE
Roadupdate oct_nov 2021

### PR DESCRIPTION
**Pull request by gregbunce**
draft pr for quality check

### CODE REVIEW

**Comment by gregbunce** [comment]
ready to go from my vantage point.

File: _posts/2021-11-09-utah-sgid-statewide-roads-data-layer-updates-11092021.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Utah SGID Statewide Roads Data Layer Updates 11/09/2021
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2021-11-09 10:30:00
+categories:
+  - Featured
+tags:
+  - Roads
```
Comment (steveoh) [inline _posts/2021-11-09-utah-sgid-statewide-roads-data-layer-updates-11092021.md:10]: **nitpick**: tags should be lowercased even though we don;t use them
```suggestion
  - roads
```

**Review by steveoh — APPROVED** [review]

File: _posts/2021-11-09-utah-sgid-statewide-roads-data-layer-updates-11092021.md (hunk 1 above)
Comment (gregbunce) [reply _posts/2021-11-09-utah-sgid-statewide-roads-data-layer-updates-11092021.md]: right on.

**Comment by gregbunce** [comment]
looks like it's waiting for an 'approve' from @steveoh .  thanks!

**Review by steveoh — APPROVED** [review]

**Pull request merged**